src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -172,6 +173,14 @@ class ConsignmentService(
       .map(cr => convertRowToConsignment(cr))
       .map(c => ConsignmentEdge(c, c.consignmentReference))
   }
+
+  private def getSeriesName(seriesId: Option[UUID]): Future[Option[String]] = {
```

Comment:
Should use this method in the `updateSeriesOfConsignment` method to reduce duplicate code.